### DISCUSSION TITLE
QUIT URL DETECTED

### DISCUSSION THREAD

**Discussion by ToniPoponi** [SafeExamBrowser/seb-win-refactoring#466] [Q&A]
Can someone please help us with this? this message keeps popping up....I am using Moodle. I have Windows 10 installed. I have used different browsers already (Chrome, Firefox, Edge). SEB version 3.4.0.480 (latest version) re-downloaded and re-installed several times....

the same error was seen even using Windows 11....what are we doing wrong? thank you!

![image](https://user-images.githubusercontent.com/113614013/190428674-327d1060-702b-4d88-a70e-cf07c05a54b6.png)

**Comment by dbuechel** [comment]
This isn't an error message, it's an information message (or rather question) specifying that SEB has detected a Quit URL (as the message itself already explains...). Might it be that you've accidentally configured the same URL as start as well as quit URL?

The documentation should explain everything you need to know with respect to configuration in general and the quit URL in particular: https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane.
